goldilocks/src/extension.rs

```diff
@@ -20,5 +23,12 @@ mod test_quadratic_extension {
 
     use p3_field_testing::test_field;
 
-    test_field!(p3_field::extension::quadratic::QuadraticBef<crate::Goldilocks>);
+    // test_field!(p3_field::extension::quadratic::QuadraticBef<crate::Goldilocks>);
```

Comment:
Will `QuadraticBef` go away now that there's more general `BinomialExtensionField`? Should we delete?

Reply:
Yes, currently, the qudraticBef and CubicBef are not in use. As (https://github.com/Plonky3/Plonky3/pull/116#issuecomment-1735581718) mentioned, we may keep it for now until (https://github.com/Plonky3/Plonky3/issues/130) is done and we can see the performance difference between the general solution and these specific optimizations and decide whether to keep them or not.